code/modules/mob/living/carbon/human/examine_vr.dm

```diff
@@ -91,7 +91,7 @@
 			message = "<span class='warning'>[t_He] [t_is] extremely hungry. A deep growl occasionally rumbles from [t_his] empty stomach.</span>\n"
 		if(100 to 499)
 			return message //Well that's pretty normal, really.
-		if(500 to 864) // Fat.
+		if(500 to 1199) // Fat.
```

Comment:
This entire code is bad. This X to Y shouldn't be how it's done. I don't remember who ported this, but they did it poorly, because using "to" doesn't work in any instance you're using decimals. Therefore this whole thing should be changed to ==, =>, and =< instead of "to" this and "to" that.